src/components/top-bar/styles.ts

```diff
@@ -34,7 +34,7 @@ export const ElTopBar = styled.div`
   border-bottom: var(--border-default, 1px) solid var(--outline-default, #e5e9ed);
   background: var(--fill-white, #fff);
 
-  ${isDesktopOrBelow} {
+  ${isBelowWideScreen} {
```

Comment:
I initially overlooked this because there are no < or > indicators in the design. However, I now believe this is correct since the only viewport with an indicator is mobile (<768px). This implies that tablet starting at 768px should mean 768px++, and similarly, desktop should be 1024px++, and so on.